src/UI/Internal/Tables/FiltersView.elm

```diff
@@ -297,7 +297,7 @@ dialog renderConfig config filter clearMsg applyMsg content =
     in
     overlay config.discardMsg <|
         Element.column
-            [ Element.width fill
+            [ Element.width <| Element.shrink
```

Comment:
How does this change affect the filters with just TextFields? Cause they're all using "width fill".
We may need a hackish solution where `min-width: 100%; width: min-content;`.
(I'm commenting this without testing, feel free to revoke If I'm wrong)

Reply:
>How does this change affect the filters with just TextFields? Cause they're all using "width fill".

This is how they look like:

![Screenshot_2020-09-01 Storybook Elm](https://user-images.githubusercontent.com/6733537/91873436-70331e00-ec92-11ea-88bd-3cb8dd2c16cc.png)

> We may need a hackish solution where min-width: 100%; width: min-content;.

is it possible to use raw css with elm-ui?

Reply:
Take a look at `UI.Internal.Utils.Element`

Reply:
> We may need a hackish solution where min-width: 100%; width: min-content;.

I think this can do the trick 🤔 

Also @segmentationfaulter I found something a bit odd and I'm not sure why exactly, but I think the size here shouldn't change right? 🤔 

![Sep-01-2020 19-02-49](https://user-images.githubusercontent.com/1559013/91884188-e5e4bd00-ec85-11ea-8e09-1621610c6836.gif)

Reply:
> Also @segmentationfaulter I found something a bit odd and I'm not sure why exactly, but I think the size here shouldn't change right?

This is happening because as soon as we make a selection, red button to clear the filter appears at the bottom and dialog makes room for it due to the rule `width: min-content`. If dialog doesn't expand in width, red button will be slightly overflowing as seen in the gif below

![dialog_overflow](https://user-images.githubusercontent.com/6733537/91958039-445e7980-ed20-11ea-922c-0da2c5a7a4ca.gif)

> We may need a hackish solution where min-width: 100%; width: min-content;

Thanks @PedroHLC for the help, let me know if we need to change where we should place `dialogWidthAttrs`

Reply:
> This is happening because as soon as we make a selection, red button to clear the filter appears at the bottom and dialog makes room for it due to the rule width: min-content. If the dialog doesn't expand in width, red button will be slightly overflowing as seen in the gif below

got it, it makes sense :D. I'm wondering if does not make sense then to have a minimum width taking into account this both button size 🤔 , this way we avoid this resize.

Also, I think your changes made the radio really small, is that intended?

![Screen Shot 2020-09-02 at 12 50 14](https://user-images.githubusercontent.com/1559013/91972593-39edb100-ed1b-11ea-952b-1bf72387890a.png)

Reply:
>I'm wondering if does not make sense then to have a minimum width taking into account this both button size

I am afraid doing so could result in both the buttons overlapping in a situation when parent column width is too narrow.

>Also, I think your changes made the radio really small, is that intended?

This was basically a side effect of the changes I made with the sizes of checkboxes and radios as per [feedback by Sufyan](https://www.figma.com/file/rqAoj2WwQZYoisP2G6d6nJ?node-id=1:41#37303481). As per this change, all the checkboxes and radio buttons will be having a size of 20x20. I have pushed a fix for selectable table checboxes size though.

Reply:
So, wouldn't it make sense to remove `Checkbox.withSize` ?

Reply:
>So, wouldn't it make sense to remove Checkbox.withSize ?

Yes, it makes sense i think, should I go with it?

Reply:
@PedroHLC I have removed `Checkbox.withSize` in the latest commit, if we think they should be retained, i will revert the commit.

cc: @jouderianjr